src/index.js

```diff
@@ -0,0 +1,49 @@
+import Navbar from './navbar';
+import Content from './content';
+import Home from './home';
+// import menu from './menu'
+// import { doc } from "prettier";
```

Comment:
- [ ] Please remove commented code. You can use git history if you need to reviwew previous versions of your codebase.